features/commenting_an_order_by_an_administrator.feature

```diff
@@ -38,4 +38,10 @@ Feature: Commenting an order by an administrator
     @application
     Scenario: Sending an email notification to the customer about unread comments
         Given I have commented the order "#00000022" with "How can I help you?"
+        And the notify customer checkbox is enabled
         Then the notification email should be sent to the customer about "How can I help you?" comment
+
+    @application
+    Scenario: Sending no email notification to the customer about unread comments
+        Given I have commented the order "#00000022" with "How can I help you?"
```

Comment:
Instead of the `Given` or the `Then`?

Reply:
The general idea is, we should always follow `Given` -> `When` -> `Then` schema. When we have only 2 steps, it's good to design a scenario to use `When` and `Then` (in fact, we almost always have some `Given`s in the background). 
But we should also remember, that a specific keyword should always come with a specific form of the sentence. In this case, `Given I have commented...` is correct from a semantical point of view, but because it's followed by `Then the notification email should not be sent...` it would be better to express it like:

```
When I comment the order "#00000022"...
```
Such a simple case, such a long comment, sorry for that 😄

Reply:
I think I fixed it.